dependencies/src/test/resources/exclusion-sample-rules/target-package.xml

```diff
@@ -0,0 +1,7 @@
+<LinkageCheckerFilter>
+  <LinkageError>
+    <Target>
+      <Package name="com.google" />
```

Comment:
does this include subclasses?

Reply:
Yes, it includes subpackages, as we do the special cases in [Linkage Checker](https://github.com/GoogleCloudPlatform/cloud-opensource-java/blob/master/dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageChecker.java#L208)